src/worker.py

```
@@ -808,21 +971,36 @@ async def api_register(req, env):
+    v_token    = generate_secure_token()
+    v_hash     = hash_token(v_token)
+    v_id       = new_id()
+    expires_at = (
+        datetime.datetime.utcnow() + datetime.timedelta(hours=24)
+    ).strftime("%Y-%m-%d %H:%M:%S")
+    try:
+        await env.DB.prepare(
+            "INSERT INTO email_verification_tokens (id,user_id,token_hash,expires_at)"
+            " VALUES (?,?,?,?)"
+        ).bind(v_id, uid, v_hash, expires_at).run()
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)